docs/tomcat.md

```diff
@@ -47,3 +47,70 @@ In addition to Tomcat specific metrics, by default Tomcat will also report [JVM
 | workload.googleapis.com/tomcat.request_count         | Cumulative     | requests    | proto_handler                  | The total requests. |
 
 
+
+# `tomcat_system` and `tomcat_access` Logging Receivers
+
+## Configuration
+
+To configure a receiver for your tomcat system logs, specify the following fields:
+
+| Field                 | Default                           | Description |
+| ---                   | ---                               | ---         |
+| `type`                | required                          | Must be `tomcat_system`. |
+| `include_paths`       | `[/opt/tomcat/logs/catalina.out]` | A list of filesystem paths to read by tailing each file. A wild card (`*`) can be used in the paths; for example, `/var/log/apache*/*.log`.
+| `exclude_paths`       | `[]`                              | A list of filesystem path patterns to exclude from the set matched by `include_paths`.
+
+To configure a receiver for your tomcat access logs, specify the following fields:
+
+| Field                 | Default                                         | Description |
+| ---                   | ---                                             | ---         |
+| `type`                | required                                        | Must be `tomcat_access`. |
+| `include_paths`       | `[/opt/tomcat/logs/localhost_access_log.*.txt]` | The log files to read. |
+| `exclude_paths`       | `[]`                                            | Log files to exclude (if `include_paths` contains a glob or directory). |
+
+Example Configuration:
+
+```yaml
+logging:
+  receivers:
+    tomcat_system:
+      type: tomcat_system
+    tomcat_access:
+      type: tomcat_access
+  service:
+    pipelines:
+      tomcat:
+        receivers:
+          - tomcat_system
+          - tomcat_access
+```
+
+## Logs
+
+System logs contain the following fields in the [`LogEntry`](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry):
+
+| Field | Type | Description |
+| ---   | ---- | ----------- |
+| `timestamp` | string ([`Timestamp`](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp)) | Time the request was received |
+| `jsonPayload.module` | string | Module of tomcat where the log originated |
+| `jsonPayload.level` | string | Log entry level |
```

Comment:
Remove this line as well now that it's removed from the implementation? (aka the `level` field)

Reply:
removed